### ISSUE TITLE
How to select hyperparameters so that our search results are aligned with what people wanted [in a CEV-way]?

### ISSUE THREAD

**Issue by sergia-ch** [tournesol-app/tournesol#148]
### Issue prerequisites
- CEV=Coherent Extrapolated Volition, see https://www.lesswrong.com/tag/coherent-extrapolated-volition
- you have read the Overleaf describing the current version of the loss function, ask @lenhoanglnh. tldr: there are three terms in the loss. the main term of the loss describes how the individual scores behave in case if the comparisons change. this is achieved by (for example) taking `softplus(rating * (score_left - score_right))`. the second one brings the individual scores closer to the aggregated scores, and the third one is for regularization.

### Issue description

Right now we have two crucial hyperparameters in the system

- lambda: the bigger lambda, the closer the individual preferences are to the aggregate preferences
- mu: regularizes the aggregate scores to be close to 1
  - mu is required for the scores to be in the specific range, otherwise they are translation-invariant
  - side effect: the range of the scores determines in which part of the non-linear function for the main term of the loss (softmax in the first version) the arguments are. 

The aggregate score ranking are quite different for different parameters

![image](https://user-images.githubusercontent.com/1012270/119513801-0f89cb00-bd64-11eb-9ffe-f310ba695b1f.png)

I was looking also at my own video ratings, and there seems to be a quite significant effect of hyperparameters on the scores:
- some sensitive videos sometimes gets to the top, and sometimes it does not -- this is quite important for sensitive topics if they are actually at the top or not
- in very rare extreme cases (when there's very few ratings, like on my account :) one of the worst videos becomes one of the best ones

![image](https://user-images.githubusercontent.com/1012270/119513915-28927c00-bd64-11eb-9aa6-75c7e2bbee59.png)

Going to the question in AI Ethics on whether "the problem is in the data, or the problem is in the algorithm", for me now it is clear that even such a thing as a wrong *hyperparameter of an algorithm* can lead to outcomes that some might find offensive and unacceptable...

-------

Research direction:

Regardless of the particular effect of the hyperparameters on the scores, it is generally unclear how to set them -- and we might have drastically different results every time! If such a system is running on a large scale, the choice of hyperparameters potentially becomes crucial to showing our website visitors what they actually wanted to find... How to set the hyperparameters for an open-source recommendation system?

Potential ideas:
- custom versions for those who don't like the default hyperparameters like in #128 
- voting on hyperparameters (each person votes for a number, and they are aggregated together)
